messaging/client.go

```diff
@@ -20,19 +20,24 @@ import (
 const (
 	// DefaultReconnectTimeout is the default time to wait between when a broker
 	// stream disconnects and another connection is retried.
-	DefaultReconnectTimeout = 1000 * time.Millisecond
+	DefaultReconnectTimeout = 1 * time.Second
 
 	// DefaultPingInterval is the default time to wait between checks to the broker.
-	DefaultPingInterval = 1000 * time.Millisecond
+	DefaultPingInterval = 1 * time.Second
+
+	// DefaultHeartbeatInterval is the default time that a topic subscriber heartbeats
+	// with a broker
+	DefaultHeartbeatInterval = 1 * time.Second
 )
 
 // Client represents a client for the broker's HTTP API.
 type Client struct {
-	mu    sync.Mutex
-	path  string    // config file path
-	conns []*Conn   // all connections opened by client
-	url   url.URL   // current known leader URL
-	urls  []url.URL // list of available broker URLs
+	mu      sync.Mutex
+	path    string    // config file path
+	conns   []*Conn   // all connections opened by client
+	url     url.URL   // current known leader URL
```

Comment:
This was there before, but we need to come back and fix this naming. `url` and `urls` are not great names on the same object. As names they don't mean much, and are too similar. They need to become `leaderURL` and `brokerURLs`. 

Not something required for this PR, one of us can fix it up afterwards.